scooby/__main__.py

```diff
@@ -59,32 +59,56 @@ def act(args_dict: Dict[str, Any]) -> None:
         print(f"scooby v{scooby.__version__}")
         return
 
-    # Report of another package.
     report = args_dict.pop('report')
+    no_opt = args_dict.pop('no_opt')
+    packages = args_dict.pop('packages')
+
+    if no_opt is None and report is None:
+        no_opt = False
+    elif no_opt is None:
+        no_opt = True
+
```

Comment:
if no_opt is None:
        if report is None:
            no_opt = False
        else:
            no_opt = True

[Not sure which one I prefer, choose your preference.]

Reply:
Much more clear and understandable